main.tf

```
@@ -41,11 +41,12 @@ resource "aws_vpc" "default" {
   ipv6_netmask_length = try(var.ipv6_primary_cidr_block_association.ipv6_netmask_length, null)
   # Additional IPv6 CIDRs are handled by aws_vpc_ipv6_cidr_block_association below
 
-  instance_tenancy                 = var.instance_tenancy
-  enable_dns_hostnames             = local.dns_hostnames_enabled
-  enable_dns_support               = local.dns_support_enabled
-  assign_generated_ipv6_cidr_block = local.assign_generated_ipv6_cidr_block
-  tags                             = module.label.tags
+  instance_tenancy                     = var.instance_tenancy
+  enable_dns_hostnames                 = local.dns_hostnames_enabled
+  enable_dns_support                   = local.dns_support_enabled
+  assign_generated_ipv6_cidr_block     = local.assign_generated_ipv6_cidr_block
+  enable_network_address_usage_metrics = var.enable_network_address_usage_metrics
```

Please use the cp convention where the `enabled` is past tense and suffixed instead of prefixed to keep arguments consistent

```suggestion
  enable_network_address_usage_metrics = var.network_address_usage_metrics_enabled
```

---

Thanks for the feedback it's done!